### ISSUE TITLE
images sampled during LORA FLUX schnell training and after training are noisy

### ISSUE THREAD

**Issue by JEFFSEVENTHSENSE** [kohya-ss/sd-scripts#1695]
dataset file:
[general]
shuffle_caption = false
caption_extension = '.txt'
keep_tokens = 1
enable_bucket = true
bucket_no_upscale = true

# This is a DreamBooth-style dataset
[[datasets]]
resolution = 512
batch_size = 1
keep_tokens = 1

  [[datasets.subsets]]
  image_dir = '/home/dluser/development/Jeff/jeff' 
  class_tokens = 'J3FF'
  num_repeats = 10

training script:
#!/bin/bash

CUDA_VISIBLE_DEVICES=0 accelerate launch \
    --mixed_precision bf16 \
    --num_cpu_threads_per_process 1 \
    flux_train_network.py \
    --pretrained_model_name_or_path flux1-schnell.safetensors \
    --clip_l /home/dluser/development/Jeff/sd-scripts/sd3/clip_l.safetensors \
    --t5xxl /home/dluser/development/Jeff/sd-scripts/sd3/t5xxl_fp16.safetensors \
    --ae ae.safetensors \
    --save_model_as safetensors \
    --sdpa \
    --persistent_data_loader_workers \
    --max_data_loader_n_workers 4 \
    --seed 42 \
    --gradient_checkpointing \
    --mixed_precision bf16 \
    --save_precision bf16 \
    --network_module networks.lora_flux \
    --network_dim 4 \
    --optimizer_type Adafactor\
    --optimizer_args scale_parameter=False relative_step=False warmup_init=False weight_decay=0.01\
    --learning_rate 1e-4 \
    --highvram \
    --max_train_epochs 10 \
    --save_every_n_epochs 4 \
    --dataset_config dataset_1024_bs2.toml \
    --output_dir /home/dluser/development/Jeff/LoRA \
    --output_name flux-lora-jeff_512_20 \
    --discrete_flow_shift 3.1582 \
    --model_prediction_type raw \
    --guidance_scale 1.0 \
    --network_train_unet_only \
    --cache_text_encoder_outputs \
    --cache_text_encoder_outputs_to_disk\
    --wandb_run_name FLUX-SCHNELL-LORA\
    --wandb_api_key f73b8bf34e217b25ca636493593fa58542db6a76\
    --sample_prompts /home/dluser/development/Jeff/sd-scripts/prompt.txt\
    --sample_every_n_steps 100\
    --log_config \
    --log_with wandb\
    --full_bf16 \

at 100 steps
![image](https://github.com/user-attachments/assets/34af192b-0a39-4e5f-a859-bb5f652f85b7)
at 200 steps 
![image](https://github.com/user-attachments/assets/f45acadd-f3d4-4ad2-871a-96a2d8459238)
at 300 steps
![image](https://github.com/user-attachments/assets/d2afd502-b73f-44df-9900-e1df5b2eb7e8)
at 400 steps
![image](https://github.com/user-attachments/assets/bb5202ba-04f8-4fe8-8807-9d9cc5d7f7ea)
 at 500 steps
![image](https://github.com/user-attachments/assets/18280195-a95b-4545-ac09-2f03296ae378)
at 600 steps
![image](https://github.com/user-attachments/assets/6f3375d3-14b6-4beb-bae3-6ba40b129858)
at 1600 steps 
![image](https://github.com/user-attachments/assets/e2cc427c-f031-447a-b508-25240bfeedfc)

is there any idea why my generated sample image are so blurry/noisy  , the result LoRA model also gives images that are of similar blurriness or noise

**Comment by waomodder** [comment]
@JEFFSEVENTHSENSE 
For reference, some of the posters here have encountered similar problems.
https://github.com/kohya-ss/sd-scripts/issues/1687#issuecomment-2406939601

**Comment by JEFFSEVENTHSENSE** [comment]
Okay i realised that currently the sd-scripts does not support schnell training. I used the training scripts from ostrisis ai-toolkit and trained a schnell model with same settings as the one mentioned. I now have better results, i am thinking because of how is schnell model trained without the training adapter compared to ai tool kit. so if anyone wants to train the schnell model , i would suggest to train it on ai-toolkit and get the LoRA and convert it to sd-scripts format to use in here.

**Comment by sp1cae** [comment]
same error
